integration-cli/docker_utils.go

```diff
@@ -679,6 +679,22 @@ func dockerCmdInDirWithTimeout(timeout time.Duration, path string, args ...strin
 	return out, status, err
 }
 
+func findContainerExitCode(c *check.C, name string, vargs ...string) (string, int) {
+	args := append(vargs, "inspect", "--format='{{ .State.ExitCode }} {{ .State.Error }}'", name)
+	cmd := exec.Command(dockerBinary, args...)
+	out, _, err := runCommandWithOutput(cmd)
+	if err != nil {
+		c.Fatal(err, out)
+	}
+	// return 127 for Windows build testing
```

Comment:
I'm still really confused here. I did not see a difference between a Linux CLI and a Windows CLI pointing to a Linux daemon. On both I ran docker run busybox foo, and both came back with State.ExitCode=-1 when run on master.